[2013-06-29 10:11:06] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /job/sensio-labs/paris-france/8/web-developer-expired/"" at C:\BitNami\wampstack-5.4.13-2\apache2\htdocs\Symfony\vendor\symfony\symfony\src\Symfony\Component\HttpKernel\EventListener\RouterListener.php line 116 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /job/sensio-labs/paris-france/8/web-developer-expired/\" at C:\\BitNami\\wampstack-5.4.13-2\\apache2\\htdocs\\Symfony\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\HttpKernel\\EventListener\\RouterListener.php:116, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\BitNami\\wampstack-5.4.13-2\\apache2\\htdocs\\Symfony\\app\\cache\\prod\\appProdUrlMatcher.php:93)"} []
[2013-07-23 12:52:50] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /"" at C:\BitNami\wampstack-5.4.13-2\apache2\htdocs\Symfony\app\cache\prod\classes.php line 1871 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /\" at C:\\BitNami\\wampstack-5.4.13-2\\apache2\\htdocs\\Symfony\\app\\cache\\prod\\classes.php:1871, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\BitNami\\wampstack-5.4.13-2\\apache2\\htdocs\\Symfony\\app\\cache\\prod\\appProdUrlMatcher.php:93)"} []
[2013-07-23 12:52:50] request.ERROR: Exception thrown when handling an exception (Twig_Error_Loader: The "C:\BitNami\wampstack-5.4.13-2\apache2\htdocs\Symfony\src\Ens\JobeetBundle/Resources/views" directory does not exist.) {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /\" at C:\\BitNami\\wampstack-5.4.13-2\\apache2\\htdocs\\Symfony\\app\\cache\\prod\\classes.php:1871, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\BitNami\\wampstack-5.4.13-2\\apache2\\htdocs\\Symfony\\app\\cache\\prod\\appProdUrlMatcher.php:93)"} []
[2013-08-13 13:44:11] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /"" at C:\BitNami\wampstack-5.4.13-2\apache2\htdocs\Symfony\app\cache\prod\classes.php line 1871 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /\" at C:\\BitNami\\wampstack-5.4.13-2\\apache2\\htdocs\\Symfony\\app\\cache\\prod\\classes.php:1871, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\BitNami\\wampstack-5.4.13-2\\apache2\\htdocs\\Symfony\\app\\cache\\prod\\appProdUrlMatcher.php:93)"} []
[2013-08-13 13:44:11] request.ERROR: Exception thrown when handling an exception (Twig_Error_Loader: The "C:\BitNami\wampstack-5.4.13-2\apache2\htdocs\Symfony\src\Ens\JobeetBundle/Resources/views" directory does not exist.) {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /\" at C:\\BitNami\\wampstack-5.4.13-2\\apache2\\htdocs\\Symfony\\app\\cache\\prod\\classes.php:1871, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\BitNami\\wampstack-5.4.13-2\\apache2\\htdocs\\Symfony\\app\\cache\\prod\\appProdUrlMatcher.php:93)"} []
[2013-08-13 13:44:20] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /admin"" at C:\BitNami\wampstack-5.4.13-2\apache2\htdocs\Symfony\app\cache\prod\classes.php line 1871 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /admin\" at C:\\BitNami\\wampstack-5.4.13-2\\apache2\\htdocs\\Symfony\\app\\cache\\prod\\classes.php:1871, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\BitNami\\wampstack-5.4.13-2\\apache2\\htdocs\\Symfony\\app\\cache\\prod\\appProdUrlMatcher.php:93)"} []
[2013-08-13 13:44:20] request.ERROR: Exception thrown when handling an exception (Twig_Error_Loader: The "C:\BitNami\wampstack-5.4.13-2\apache2\htdocs\Symfony\src\Ens\JobeetBundle/Resources/views" directory does not exist.) {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /admin\" at C:\\BitNami\\wampstack-5.4.13-2\\apache2\\htdocs\\Symfony\\app\\cache\\prod\\classes.php:1871, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\BitNami\\wampstack-5.4.13-2\\apache2\\htdocs\\Symfony\\app\\cache\\prod\\appProdUrlMatcher.php:93)"} []
[2013-08-13 14:04:12] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /"" at C:\BitNami\wampstack-5.4.13-2\apache2\htdocs\Symfony\app\cache\prod\classes.php line 1871 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /\" at C:\\BitNami\\wampstack-5.4.13-2\\apache2\\htdocs\\Symfony\\app\\cache\\prod\\classes.php:1871, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\BitNami\\wampstack-5.4.13-2\\apache2\\htdocs\\Symfony\\app\\cache\\prod\\appProdUrlMatcher.php:93)"} []
[2013-08-13 14:04:12] request.ERROR: Exception thrown when handling an exception (Twig_Error_Loader: The "C:\BitNami\wampstack-5.4.13-2\apache2\htdocs\Symfony\src\Ens\JobeetBundle/Resources/views" directory does not exist.) {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /\" at C:\\BitNami\\wampstack-5.4.13-2\\apache2\\htdocs\\Symfony\\app\\cache\\prod\\classes.php:1871, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\BitNami\\wampstack-5.4.13-2\\apache2\\htdocs\\Symfony\\app\\cache\\prod\\appProdUrlMatcher.php:93)"} []
[2013-08-13 14:04:18] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /"" at C:\BitNami\wampstack-5.4.13-2\apache2\htdocs\Symfony\app\cache\prod\classes.php line 1871 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /\" at C:\\BitNami\\wampstack-5.4.13-2\\apache2\\htdocs\\Symfony\\app\\cache\\prod\\classes.php:1871, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\BitNami\\wampstack-5.4.13-2\\apache2\\htdocs\\Symfony\\app\\cache\\prod\\appProdUrlMatcher.php:93)"} []
[2013-08-13 14:04:18] request.ERROR: Exception thrown when handling an exception (Twig_Error_Loader: The "C:\BitNami\wampstack-5.4.13-2\apache2\htdocs\Symfony\src\Ens\JobeetBundle/Resources/views" directory does not exist.) {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /\" at C:\\BitNami\\wampstack-5.4.13-2\\apache2\\htdocs\\Symfony\\app\\cache\\prod\\classes.php:1871, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\BitNami\\wampstack-5.4.13-2\\apache2\\htdocs\\Symfony\\app\\cache\\prod\\appProdUrlMatcher.php:93)"} []
[2013-08-13 14:10:33] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /admin/fuentedefinanciamiento/new"" at C:\BitNami\wampstack-5.4.13-2\apache2\htdocs\Symfony\app\cache\prod\classes.php line 1871 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /admin/fuentedefinanciamiento/new\" at C:\\BitNami\\wampstack-5.4.13-2\\apache2\\htdocs\\Symfony\\app\\cache\\prod\\classes.php:1871, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\BitNami\\wampstack-5.4.13-2\\apache2\\htdocs\\Symfony\\app\\cache\\prod\\appProdUrlMatcher.php:93)"} []
[2013-08-13 14:10:33] request.ERROR: Exception thrown when handling an exception (Twig_Error_Loader: The "C:\BitNami\wampstack-5.4.13-2\apache2\htdocs\Symfony\src\Ens\JobeetBundle/Resources/views" directory does not exist.) {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /admin/fuentedefinanciamiento/new\" at C:\\BitNami\\wampstack-5.4.13-2\\apache2\\htdocs\\Symfony\\app\\cache\\prod\\classes.php:1871, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\BitNami\\wampstack-5.4.13-2\\apache2\\htdocs\\Symfony\\app\\cache\\prod\\appProdUrlMatcher.php:93)"} []
[2013-08-13 14:10:35] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /admin/fuentedefinanciamiento/new"" at C:\BitNami\wampstack-5.4.13-2\apache2\htdocs\Symfony\app\cache\prod\classes.php line 1871 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /admin/fuentedefinanciamiento/new\" at C:\\BitNami\\wampstack-5.4.13-2\\apache2\\htdocs\\Symfony\\app\\cache\\prod\\classes.php:1871, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\BitNami\\wampstack-5.4.13-2\\apache2\\htdocs\\Symfony\\app\\cache\\prod\\appProdUrlMatcher.php:93)"} []
[2013-08-13 14:10:35] request.ERROR: Exception thrown when handling an exception (Twig_Error_Loader: The "C:\BitNami\wampstack-5.4.13-2\apache2\htdocs\Symfony\src\Ens\JobeetBundle/Resources/views" directory does not exist.) {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /admin/fuentedefinanciamiento/new\" at C:\\BitNami\\wampstack-5.4.13-2\\apache2\\htdocs\\Symfony\\app\\cache\\prod\\classes.php:1871, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\BitNami\\wampstack-5.4.13-2\\apache2\\htdocs\\Symfony\\app\\cache\\prod\\appProdUrlMatcher.php:93)"} []
[2013-08-13 14:10:40] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /admin/fuentedefinanciamiento/new"" at C:\BitNami\wampstack-5.4.13-2\apache2\htdocs\Symfony\app\cache\prod\classes.php line 1871 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /admin/fuentedefinanciamiento/new\" at C:\\BitNami\\wampstack-5.4.13-2\\apache2\\htdocs\\Symfony\\app\\cache\\prod\\classes.php:1871, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\BitNami\\wampstack-5.4.13-2\\apache2\\htdocs\\Symfony\\app\\cache\\prod\\appProdUrlMatcher.php:93)"} []
[2013-08-13 14:10:40] request.ERROR: Exception thrown when handling an exception (Twig_Error_Loader: The "C:\BitNami\wampstack-5.4.13-2\apache2\htdocs\Symfony\src\Ens\JobeetBundle/Resources/views" directory does not exist.) {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /admin/fuentedefinanciamiento/new\" at C:\\BitNami\\wampstack-5.4.13-2\\apache2\\htdocs\\Symfony\\app\\cache\\prod\\classes.php:1871, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\BitNami\\wampstack-5.4.13-2\\apache2\\htdocs\\Symfony\\app\\cache\\prod\\appProdUrlMatcher.php:93)"} []
[2013-08-13 14:10:40] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /admin/fuentedefinanciamiento/new"" at C:\BitNami\wampstack-5.4.13-2\apache2\htdocs\Symfony\app\cache\prod\classes.php line 1871 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /admin/fuentedefinanciamiento/new\" at C:\\BitNami\\wampstack-5.4.13-2\\apache2\\htdocs\\Symfony\\app\\cache\\prod\\classes.php:1871, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\BitNami\\wampstack-5.4.13-2\\apache2\\htdocs\\Symfony\\app\\cache\\prod\\appProdUrlMatcher.php:93)"} []
[2013-08-13 14:10:40] request.ERROR: Exception thrown when handling an exception (Twig_Error_Loader: The "C:\BitNami\wampstack-5.4.13-2\apache2\htdocs\Symfony\src\Ens\JobeetBundle/Resources/views" directory does not exist.) {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /admin/fuentedefinanciamiento/new\" at C:\\BitNami\\wampstack-5.4.13-2\\apache2\\htdocs\\Symfony\\app\\cache\\prod\\classes.php:1871, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\BitNami\\wampstack-5.4.13-2\\apache2\\htdocs\\Symfony\\app\\cache\\prod\\appProdUrlMatcher.php:93)"} []
[2013-08-13 14:10:40] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /admin/fuentedefinanciamiento/new"" at C:\BitNami\wampstack-5.4.13-2\apache2\htdocs\Symfony\app\cache\prod\classes.php line 1871 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /admin/fuentedefinanciamiento/new\" at C:\\BitNami\\wampstack-5.4.13-2\\apache2\\htdocs\\Symfony\\app\\cache\\prod\\classes.php:1871, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\BitNami\\wampstack-5.4.13-2\\apache2\\htdocs\\Symfony\\app\\cache\\prod\\appProdUrlMatcher.php:93)"} []
[2013-08-13 14:10:40] request.ERROR: Exception thrown when handling an exception (Twig_Error_Loader: The "C:\BitNami\wampstack-5.4.13-2\apache2\htdocs\Symfony\src\Ens\JobeetBundle/Resources/views" directory does not exist.) {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /admin/fuentedefinanciamiento/new\" at C:\\BitNami\\wampstack-5.4.13-2\\apache2\\htdocs\\Symfony\\app\\cache\\prod\\classes.php:1871, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException:  at C:\\BitNami\\wampstack-5.4.13-2\\apache2\\htdocs\\Symfony\\app\\cache\\prod\\appProdUrlMatcher.php:93)"} []
[2013-11-27 17:57:31] request.INFO: Matched route "uta_seg_pro_homepage" (parameters: "_controller": "UTA\SegProBundle\Controller\HomeController::indexAction", "_route": "uta_seg_pro_homepage") [] []
[2013-11-27 17:57:31] security.DEBUG: Read SecurityContext from the session [] []
[2013-11-27 17:57:31] security.DEBUG: Reloading user from user provider. [] []
[2013-11-27 17:57:31] security.DEBUG: Username "" was reloaded from user provider. [] []
[2013-11-27 17:57:31] request.CRITICAL: Uncaught PHP Exception Twig_Error_Runtime: "An exception has been thrown during the rendering of a template ("The menu "breadcrumb" is not defined.") in "UTASegProBundle:Home:index.html.twig"." at /var/www/Symfony/app/cache/prod/classes.php line 4402 {"exception":"[object] (Twig_Error_Runtime: An exception has been thrown during the rendering of a template (\"The menu \"breadcrumb\" is not defined.\") in \"UTASegProBundle:Home:index.html.twig\". at /var/www/Symfony/app/cache/prod/classes.php:4402, InvalidArgumentException: The menu \"breadcrumb\" is not defined. at /var/www/Symfony/vendor/knplabs/knp-menu/src/Knp/Menu/Provider/ChainProvider.php:25)"} []
[2013-11-27 17:57:31] security.DEBUG: Write SecurityContext in the session [] []
[2013-11-27 17:57:45] request.INFO: Matched route "uta_seg_pro_homepage" (parameters: "_controller": "UTA\SegProBundle\Controller\HomeController::indexAction", "_route": "uta_seg_pro_homepage") [] []
[2013-11-27 17:57:45] security.DEBUG: Read SecurityContext from the session [] []
[2013-11-27 17:57:45] security.DEBUG: Reloading user from user provider. [] []
[2013-11-27 17:57:45] security.DEBUG: Username "" was reloaded from user provider. [] []
[2013-11-27 17:57:45] request.CRITICAL: Uncaught PHP Exception Twig_Error_Runtime: "An exception has been thrown during the rendering of a template ("The menu "breadcrumb" is not defined.") in "UTASegProBundle:Home:index.html.twig"." at /var/www/Symfony/app/cache/prod/classes.php line 4402 {"exception":"[object] (Twig_Error_Runtime: An exception has been thrown during the rendering of a template (\"The menu \"breadcrumb\" is not defined.\") in \"UTASegProBundle:Home:index.html.twig\". at /var/www/Symfony/app/cache/prod/classes.php:4402, InvalidArgumentException: The menu \"breadcrumb\" is not defined. at /var/www/Symfony/vendor/knplabs/knp-menu/src/Knp/Menu/Provider/ChainProvider.php:25)"} []
[2013-11-27 17:57:45] security.DEBUG: Write SecurityContext in the session [] []
[2013-11-27 17:57:46] request.INFO: Matched route "uta_seg_pro_homepage" (parameters: "_controller": "UTA\SegProBundle\Controller\HomeController::indexAction", "_route": "uta_seg_pro_homepage") [] []
[2013-11-27 17:57:46] security.DEBUG: Read SecurityContext from the session [] []
[2013-11-27 17:57:46] security.DEBUG: Reloading user from user provider. [] []
[2013-11-27 17:57:46] security.DEBUG: Username "" was reloaded from user provider. [] []
[2013-11-27 17:57:46] request.CRITICAL: Uncaught PHP Exception Twig_Error_Runtime: "An exception has been thrown during the rendering of a template ("The menu "breadcrumb" is not defined.") in "UTASegProBundle:Home:index.html.twig"." at /var/www/Symfony/app/cache/prod/classes.php line 4402 {"exception":"[object] (Twig_Error_Runtime: An exception has been thrown during the rendering of a template (\"The menu \"breadcrumb\" is not defined.\") in \"UTASegProBundle:Home:index.html.twig\". at /var/www/Symfony/app/cache/prod/classes.php:4402, InvalidArgumentException: The menu \"breadcrumb\" is not defined. at /var/www/Symfony/vendor/knplabs/knp-menu/src/Knp/Menu/Provider/ChainProvider.php:25)"} []
[2013-11-27 17:57:46] security.DEBUG: Write SecurityContext in the session [] []
[2013-11-27 17:57:47] request.INFO: Matched route "uta_seg_pro_homepage" (parameters: "_controller": "UTA\SegProBundle\Controller\HomeController::indexAction", "_route": "uta_seg_pro_homepage") [] []
[2013-11-27 17:57:47] security.DEBUG: Read SecurityContext from the session [] []
[2013-11-27 17:57:47] security.DEBUG: Reloading user from user provider. [] []
[2013-11-27 17:57:47] security.DEBUG: Username "" was reloaded from user provider. [] []
[2013-11-27 17:57:47] request.CRITICAL: Uncaught PHP Exception Twig_Error_Runtime: "An exception has been thrown during the rendering of a template ("The menu "breadcrumb" is not defined.") in "UTASegProBundle:Home:index.html.twig"." at /var/www/Symfony/app/cache/prod/classes.php line 4402 {"exception":"[object] (Twig_Error_Runtime: An exception has been thrown during the rendering of a template (\"The menu \"breadcrumb\" is not defined.\") in \"UTASegProBundle:Home:index.html.twig\". at /var/www/Symfony/app/cache/prod/classes.php:4402, InvalidArgumentException: The menu \"breadcrumb\" is not defined. at /var/www/Symfony/vendor/knplabs/knp-menu/src/Knp/Menu/Provider/ChainProvider.php:25)"} []
[2013-11-27 17:57:47] security.DEBUG: Write SecurityContext in the session [] []
[2013-11-27 17:57:48] request.INFO: Matched route "uta_seg_pro_homepage" (parameters: "_controller": "UTA\SegProBundle\Controller\HomeController::indexAction", "_route": "uta_seg_pro_homepage") [] []
[2013-11-27 17:57:48] security.DEBUG: Read SecurityContext from the session [] []
[2013-11-27 17:57:48] security.DEBUG: Reloading user from user provider. [] []
[2013-11-27 17:57:48] security.DEBUG: Username "" was reloaded from user provider. [] []
[2013-11-27 17:57:48] request.CRITICAL: Uncaught PHP Exception Twig_Error_Runtime: "An exception has been thrown during the rendering of a template ("The menu "breadcrumb" is not defined.") in "UTASegProBundle:Home:index.html.twig"." at /var/www/Symfony/app/cache/prod/classes.php line 4402 {"exception":"[object] (Twig_Error_Runtime: An exception has been thrown during the rendering of a template (\"The menu \"breadcrumb\" is not defined.\") in \"UTASegProBundle:Home:index.html.twig\". at /var/www/Symfony/app/cache/prod/classes.php:4402, InvalidArgumentException: The menu \"breadcrumb\" is not defined. at /var/www/Symfony/vendor/knplabs/knp-menu/src/Knp/Menu/Provider/ChainProvider.php:25)"} []
[2013-11-27 17:57:48] security.DEBUG: Write SecurityContext in the session [] []
[2013-11-27 17:57:49] request.INFO: Matched route "uta_seg_pro_homepage" (parameters: "_controller": "UTA\SegProBundle\Controller\HomeController::indexAction", "_route": "uta_seg_pro_homepage") [] []
[2013-11-27 17:57:49] security.DEBUG: Read SecurityContext from the session [] []
[2013-11-27 17:57:49] security.DEBUG: Reloading user from user provider. [] []
[2013-11-27 17:57:49] security.DEBUG: Username "" was reloaded from user provider. [] []
[2013-11-27 17:57:49] request.CRITICAL: Uncaught PHP Exception Twig_Error_Runtime: "An exception has been thrown during the rendering of a template ("The menu "breadcrumb" is not defined.") in "UTASegProBundle:Home:index.html.twig"." at /var/www/Symfony/app/cache/prod/classes.php line 4402 {"exception":"[object] (Twig_Error_Runtime: An exception has been thrown during the rendering of a template (\"The menu \"breadcrumb\" is not defined.\") in \"UTASegProBundle:Home:index.html.twig\". at /var/www/Symfony/app/cache/prod/classes.php:4402, InvalidArgumentException: The menu \"breadcrumb\" is not defined. at /var/www/Symfony/vendor/knplabs/knp-menu/src/Knp/Menu/Provider/ChainProvider.php:25)"} []
[2013-11-27 17:57:49] security.DEBUG: Write SecurityContext in the session [] []
[2013-11-27 18:09:45] request.INFO: Matched route "uta_seg_pro_homepage" (parameters: "_controller": "UTA\SegProBundle\Controller\HomeController::indexAction", "_route": "uta_seg_pro_homepage") [] []
[2013-11-27 18:09:45] security.DEBUG: Read SecurityContext from the session [] []
[2013-11-27 18:09:45] security.DEBUG: Reloading user from user provider. [] []
[2013-11-27 18:09:45] security.DEBUG: Username "" was reloaded from user provider. [] []
[2013-11-27 18:09:45] request.CRITICAL: Uncaught PHP Exception Twig_Error_Runtime: "An exception has been thrown during the rendering of a template ("The menu "breadcrumb" is not defined.") in "UTASegProBundle:Home:index.html.twig"." at /var/www/Symfony/app/cache/prod/classes.php line 4402 {"exception":"[object] (Twig_Error_Runtime: An exception has been thrown during the rendering of a template (\"The menu \"breadcrumb\" is not defined.\") in \"UTASegProBundle:Home:index.html.twig\". at /var/www/Symfony/app/cache/prod/classes.php:4402, InvalidArgumentException: The menu \"breadcrumb\" is not defined. at /var/www/Symfony/vendor/knplabs/knp-menu/src/Knp/Menu/Provider/ChainProvider.php:25)"} []
[2013-11-27 18:09:45] security.DEBUG: Write SecurityContext in the session [] []
